doc/users/whats_new/imshow_logscales.rst

```diff
@@ -0,0 +1,31 @@
+Log-scale on image plots
+------------------------
+
+:func:`imshow` now properly displays data on log-scales.
```

Comment:
"Properly" and "correct" below don't really help describing the behavior.  Something like "draws values at the requested points in data space after application of the scale"?